packages/eleventy-plugin-cloudflare-image-resizing/test/build_cloudflare_image.spec.ts

```diff
@@ -23,6 +26,35 @@ const mockDataCloudflare: Record<
 		rebasedOriginalURL: 'cloudflare-images/car.b62406a0fe1.jpg',
 		mode: 'img',
 	},
+	imageOptionsWithCloudflareURL: {
+		normalizedZone: 'https://test.com',
+		normalizedDomain: 'https://test.com/',
+		isLocal: false,
+		fullOptions: {
+			anim: true,
+			dpr: 1,
+			format: 'auto',
+			quality: 85,
+		},
+		rebasedOriginalURL: 'cloudflare-images/car.b62406a0fe1.jpg',
+		mode: 'img',
+		cloudflareURL: (zone, domain, options, originalURL) =>
+			'http://localhost:8787' +
```

Comment:
Make a function to return a different URL compared to the `cloudflareDefaultURL` function. We need to easily spot the difference.